lib/tweet.rb

```diff
@@ -41,6 +51,14 @@ def save
     self
   end
 
+  def update_tweet_message_in_database
+    sql = <<-SQL
+    UPDATE tweets SET message = (?) WHERE id = (?)
```

Comment:
we probably also want to update the username, since right now we don't have anything preventing a user from changing it. If I didn't want a user to be able to change their username once it's, I'd put that logic somewhere else.